apps/root/src/frame/components/navigation/index.tsx

```diff
@@ -38,6 +37,7 @@ import { useThemeMode } from '@state/config/hooks';
 import useSelectedLanguage from '@hooks/useSelectedLanguage';
 import { SUPPORTED_LANGUAGES_STRING, SupportedLanguages } from '@constants/lang';
 import useChangeLanguage from '@hooks/useChangeLanguage';
+import { LangIcon } from 'ui-library/src';
```

Comment:
This import is incorrect, it should be on the top

Reply:
Yikes, autoimport why u do dis. Ty.